superduperdb/container/listener.py

```diff
@@ -48,6 +48,9 @@ def on_create(self, db: DB) -> None:
         if isinstance(self.model, str):
             self.model = t.cast(Model, db.load('model', self.model))
 
+        # Start cdc service if enabled
+        db.cdc.add(self.select.table_or_collection)
```

Comment:
Should this not depend on some kind of configuration?
E.g. 

`if self.select.table_or_collection in CFG.cdc.table_or_collections:...`

Reply:
As we discussed, 

when a listener is added to db, it adds the collection of cdc.

now if cdc is on, the above collection is automatically triggered for cdc.

Reply:
Got it.